code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -82,13 +101,19 @@
 
 /obj/item/projectile/bullet/a762
 	damage = 20
-	armor_penetration = 15
+	armor_penetration = 10
 	penetrating = 1
 	can_ricochet = TRUE
 
+/obj/item/projectile/bullet/a762h
```

Comment:
For these high velocity rounds, we should establish a standard

I'm thinking 20% higher, in both damage and AP
So give these ones 24 damage and 18 AP

this leaves a possibility open for hollowpoints and AP rounds to be added later which specialise in only one of these attributes